app/views/posts/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>New Post</h1>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214760470-7aa0b9d5-16bd-4152-ba6a-8c9787ccaa3a.png)

**[Optional]**

- [ ] Kindly remove this description to make your readme file look clean and professional :blush: